src/App.tsx

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  message: string;
+};
+
+export class App extends React.Component {
+  state: Readonly<State> = {
+    message: 'Nothing was pressed yet',
```

Comment:
in state you need to store pressed key, not message
show the message if nothing is pressed directly in render method